actors/runtime/src/test_utils.rs

```diff
@@ -1099,6 +1072,10 @@ impl Syscalls for MockRuntime {
         );
         exp.result
     }
+
+    fn on_submit_verify_seal(&self) {
+        todo!()
```

Comment:
Please implement this. I came through to implement all the necessary mock expectations just last week.